lib/artist.rb

```diff
@@ -0,0 +1,39 @@
+class Artist
+  extend Concerns::Findable
+
+  attr_accessor :name, :songs
+
+  @@all = []
+
+  def initialize(name)
+    @name = name
+    @songs = []
+  end
+
+  def self.all
+    @@all
+  end
+
+  def self.destroy_all
+    @@all.clear
+  end
+
+  def save
+    @@all << self
+  end
+
+  def self.create(name)
+    artist = Artist.new(name)
+    artist.save
+    artist
```

Comment:
You may want your `save` method to return the instance it is getting called on. That way, this method could just turn into `new(name).save`